Worksheet_D/oxo.py

```diff
@@ -0,0 +1,115 @@
+class OxoBoard:
+    def __init__(self):
+        """ The initialiser. Initialise any fields you need here. """
+        #This command sets up the gameboard as 9 different values each beginning with 0(empty)
+        self.game_board = [0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Using list multiplication, this could be written in a neater way:
```
self.game_board = [0] * 9
```
which sets `self.game_board` to be 9 copies of the single-element list `[0]` concatenated together.